test/observe.lean

```diff
@@ -1,17 +1,21 @@
-import Mathlib.Tactic.Linarith
 import Mathlib.Data.Nat.Prime
 
 open Nat
 
-set_option maxHeartbeats 8000 in
+-- Adaptation note: at nightly-2024-03-27,
+-- we had to increase `maxHeartbeats` here from 8000 to 16000.
+-- Adaptation note: at nightly-2024-04-01,
+-- we had to increase `maxHeartbeats` here from 16000 to 24000.
+-- (Note: successive runs are faster, because of the `exact?` cache initialisation.)
```

Comment:
Hmmm, I'm a bit sad about these. Do we have ideas about how to get the number down again?

Reply:
Unfortunately not yet. :-(